samples/metadata-repo-integration/build.gradle

```diff
@@ -66,3 +67,9 @@ graalvmNative {
         enabled = true
     }
 }
+
+task reachabilityMetadata(type: org.graalvm.buildtools.gradle.tasks.ReachabilityMetadataCopyTask)
```

Comment:
:x: Same comment here: both the task creation and the jar task configuration should use the lazy APIs.